CMSgov/design-system
# [WNMGDS-2203] Add `ds-cms-gov` package with JS and CSS

## Summary

https://jira.cms.gov/browse/WNMGDS-2203

## How to test

1. Check out [ad9ddaa](https://github.com/CMSgov/design-system/pull/2367/commits/ad9ddaad04bbe604f113dfc383e8934b025bea5d)
2. `yarn build:cmsgov`
3. `cd examples/react-app`
4. `yarn install && yarn build && npx http-server` and view the page

## Checklist
